packages/beacon-node/src/node/nodejs.ts

```diff
@@ -149,6 +149,9 @@ export class BeaconNode {
 
     // start db if not already started
     await db.start();
+    // Prune hot db repos
+    // TODO: Should this call be awaited?
```

Comment:
Probably so unless we can be sure the pruning is done by the time we're syncing.